qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -130,9 +128,7 @@ def main(self):
             traceback.print_tb(sys.exc_info()[2])
 
         if not self.options.noshutdown:
-            print("Stopping nodes")
```

Comment:
This gets printed out when stop_nodes() is called.  No need to print it 
out twice.

On 20/08/2015 8:56 AM, Jonas Schnelli wrote:
>
> In qa/rpc-tests/test_framework/test_framework.py 
> <https://github.com/bitcoin/bitcoin/pull/6548#discussion_r37546694>:
>
> > @@ -130,9 +128,7 @@ def main(self):
> >              traceback.print_tb(sys.exc_info()[2])
> >
> >          if not self.options.noshutdown:
> > -            print("Stopping nodes")
>
> What's the reason of removing this line?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/bitcoin/bitcoin/pull/6548/files#r37546694>.
>